app/controllers/tanks_controller.rb

```diff
@@ -1,12 +1,22 @@
 class TanksController < ApplicationController
-  before_action :set_tank, only: [:show]
+  before_action :set_tank, only: [:show, :destroy]
 
   def index
     @tanks = Tank.all
   end
 
   def show; end
 
+  # DELETE /tanks/1
+  # DELETE /tanks/1.json
+  def destroy
+    @tank.destroy
+    respond_to do |format|
+      format.html { redirect_to tanks_url, notice: 'Tank was successfully destroyed.' }
+      format.json { head :no_content }
```

Comment:
I dont think we actually _need_ json responses. No need to delete it here, as we can clean it up later with the rest of the actions that will need this cleanup.